quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -211,8 +211,9 @@ public Map<String, String> getConfigOverrides() {
   @Inject PolarisDiagnostics diagServices;
   @Inject PolarisEventListener polarisEventListener;
 
+  protected String realmName;
```

Comment:
`protected` fields are hard to refactor. Non-final shared fields are also hard to reason about (in general). Would you mind adding a `protected` getter for a `private` field instead?